OneCommunityGlobal/HighestGoodNetworkApp
# Luis copy button weekly summary

# Description
<img width="991" alt="Screenshot 2023-08-22 at 9 16 14 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/57dc98a9-3383-462c-9134-383f306cea2b">


## Main changes explained:

Added a copy icon to user's profile component. When viewing weekly summaries admins can quickly copy and paste their team members summaries to speed up their managerial tasks. 
Jae also suggested to add the icon to the weekly reports summary to view everyone's weekly summaries to counter-act the current bug. This way admins can copy weekly summaries there as well. 

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin user or owner.
4. go to dashboard→ view profile → show weekly summary → select a team member.
5. View their weekly summaries and click on the copy button to copy their summary to your clipboard. 
<img width="899" alt="Screenshot 2023-08-22 at 9 23 55 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/ef105e5c-5f67-4c24-a2b5-450d9da500ab">


7. Verify clipboard has the exact text copied to your clipboard. 
<img width="821" alt="Screenshot 2023-08-22 at 9 11 15 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/66aeb2cc-ea7b-4a4c-82e4-7c5dd0015755">

8. Repeat from steps 3  but navigate to  dashboard→ reports → weekly summary reports  → select a team member.
9. View their weekly summaries and click on the copy button to copy their summary to your clipboard. 
<img width="692" alt="Screenshot 2023-08-22 at 9 15 21 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/457094b7-be8d-4fc2-897a-77337744632a">



## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/22454953/b13ec2c9-40a5-4fa5-b24d-c0611bb02ec3
